modules/industry/scripts/utils/jrc_idees_parser.py

```diff
@@ -0,0 +1,109 @@
+from typing import Union
+
+import xarray as xr
+
+STANDARD_COORDS = ["cat_name", "year", "country_code", "carrier_name"]
+
+
+def ensure_standard_coordinates(ds: Union[xr.Dataset, xr.DataArray]):
+    """Remove all coordinates that do not match a predefined standard."""
+    removed_coords = [i for i in ds.coords if i not in STANDARD_COORDS]
+    removed_dims = [i for i in ds.dims if i in removed_coords]
+    if any(removed_dims):
```

Comment:
```suggestion
    removed_coords = set(ds.coords).difference(STANDARD_COORDS)
    removed_dims = removed_coords.intersection(ds.dims)
    if removed_dims: 
```
an empty set will resolve to False, otherwise True.

Is this an expected problem, that there would be dimensions with coordinates? Otherwise I'd say that the first two lines could just be `set(ds.dims).difference(STANDARD_COORDS)`

Reply:
Oh this is much more efficient! Thanks.